easybuild/easyconfigs/s/statsmodels/statsmodels-0.9.0-intel-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'PythonBundle'
+
+name = 'statsmodels'
+version = '0.9.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.statsmodels.org'
+description = """Statsmodels is a Python module that provides classes and functions
+for the estimation of many different statistical models, as well as for conducting 
+statistical tests, and statistical data exploration."""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+
+dependencies = [('Python', '2.7.15')]
+
+use_pip = True
+
+exts_list = [
+    ('patsy', '0.5.1', {
+        'source_urls': [PYPI_SOURCE],
+        'sources': ['%(name)s-%(version)s.tar.gz'],
```

Comment:
in an extension list it should be `'source_tmpl':` instead of `'sources':`, but it is only necessary if different from this format, which is the default

Reply:
thx, fixed.